Assets/Scripts/Player/PlayerController.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour {
+	[Header("Movement")]
+	[SerializeField] private float acceleration = 25;
+	[SerializeField] private float deacceleration = 5;
+	[SerializeField] private float maxSpeed = 3;
+
+	[Header("Jumping")]
+	[SerializeField] private float jumpVelocity = 7.5f;
+
+	private Rigidbody2D rb2D;
+	private BoxCollider2D boxCollider;
+	private Vector2 velocity;
+
+	private bool doJump = false;
+	private bool isGrounded = false;
+
+	public bool IsGrounded => isGrounded;
+
+	private void Start() {
+		rb2D = GetComponent<Rigidbody2D>();
+		boxCollider = GetComponent<BoxCollider2D>();
+	}
+
+	private void Update() {
+		isGrounded = CheckIfGrounded();
+		float xInput = Input.GetAxisRaw("Horizontal");
+
+		if (rb2D.velocity.x == 0)
+			velocity.x = 0;
+
+		if (xInput != 0) {
+			float newVelocityX = velocity.x + xInput * acceleration * Time.deltaTime;
+			velocity.x = Mathf.Clamp(newVelocityX, -maxSpeed, maxSpeed);
+		} else {
+			velocity.x -= velocity.x * deacceleration * Time.deltaTime;
+		}
+
+		if (Input.GetButtonDown("Jump") && isGrounded)
```

Comment:
It's cheaper to check the bool before checking the input.
```suggestion
		if (isGrounded && Input.GetButtonDown("Jump"))
```